src/pages/Autolending/AutolendingSettingsPage.vue

```diff
@@ -6,77 +6,7 @@
 		<autolending-when />
 
 		<!-- Who you'll support-->
-		<div class="row who-area" :class="{ obscure: !isEnabled }">
```

Comment:
Removal of template, components and css that now lives in the autolending-who component.